frame/nomination-pools/src/lib.rs

```diff
@@ -622,17 +622,19 @@ impl<T: Config> BondedPool<T> {
 	///
 	/// This is often used for bonding and issuing new funds into the pool.
 	fn balance_to_point(&self, new_funds: BalanceOf<T>) -> BalanceOf<T> {
-		let bonded_balance =
-			T::StakingInterface::active_stake(&self.bonded_account()).unwrap_or(Zero::zero());
+		let bonded_balance = T::Staking::stake(&self.bonded_account())
```

Comment:
I guess it is harmless to still have `fn active_balance` and `fn total_balance` on staking interface that are auto-implemented, preventing a little bit of dupe `.map`ing here.

Reply:
Yeah, I wanted to implement the extra stuff on `Stake` struct, but it'd still require mapping as we work with `Result`. So I guess we'd still be better off with bloating the interface a little.